### PR TITLE
Add a way for advertisers advertise on Guppy

### PR DESCRIPTION
**Pull request by autn**
Hi @YPCrumble, this PR is used for adding a way for advertisers to advertise on Guppy.
Please help me to review and change my Github role so I can add the secrets or help me to add the Stripe testing API keys to the secrets. Thanks!

### CODE REVIEW

**Review by YPCrumble — APPROVED** [review]
@autn the way we add secrets is to add them to the vault in our deployments repo so that they get added as environment variables to the site. Remember that the vault file should be encrypted. Truong or Ngoc should be able to show you :D.

**Comment by autn** [comment]
@YPCrumble I see, but what I want in this case is to pass the Github actions :D

**Review by YPCrumble — APPROVED** [review]

File: advertisers/models.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+    email = models.EmailField(blank=False, null=False)
+    advertisement = models.ForeignKey(
+        "advertisers.Advertisement",
+        on_delete=models.SET_NULL,
+        related_name="advertiser",
+        blank=True,
+        null=True,
+    )
+    monthly_budget = models.IntegerField(blank=False, null=False)
```
Comment (YPCrumble) [inline advertisers/models.py:24]: @autn for now let's make the monthly budget linked to an advertisement, not the advertiser.

File: advertisers/templates/advertisers/signup.html [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+{% block header_tags %}
+<title>Sign up advertiser.</title>
+<meta name="description" content="Sign up advertiser"/>
+<script src="https://js.stripe.com/v3/"></script>
+{% endblock %}
+
+{% block content %}
+<div class="row">
+  <h4 class="col-md-8 offset-md-2">Sign up advertiser</h4>
```
Comment (YPCrumble) [inline advertisers/templates/advertisers/signup.html:12]: @autn let's change this to "Create your first ad on Guppy."

**Review by YPCrumble — CHANGES_REQUESTED** [review]
@autn looking good! Just a couple comments.

**Pull request merged**